packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -70,13 +70,14 @@ class NestedScrollView extends StatefulWidget {
   /// Defaults to false.
   final bool reverse;
 
-  /// How the scroll view should respond to user input.
+  /// How the outer scroll view should respond to user input.
   ///
-  /// For example, determines how the scroll view continues to animate after the
-  /// user stops dragging the scroll view.
+  /// The physics should clamp so that the extra pixels can be given to the
+  /// inner scrollables can scroll. Passing a custom class that extends
+  /// [ClampingScrollPhysics] is allowed.
```

Comment:
`Passing a custom class that extends [ClampingScrollPhysics] is allowed.` is probably self-evident so I would omit it.
Maybe instead say something like `The value must inherit from [ClampingScrollPhysics] and its [ScrollPhysics.applyBoundaryConditions] implementation should not allow scrolling outside the scroll extent range described by the [ScrollMetrics.minScrollExtent]/[ScrollMetrics.maxScrollExtent] properties passed to that method. If that invariant is not maintained, the nested scroll view may respond to user scrolling erratically.` or some such.